app/src/main/java/de/danoeh/antennapod/fragment/PlaybackHistoryFragment.java

```diff
@@ -32,7 +33,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         emptyView.setIcon(R.drawable.ic_history);
         emptyView.setTitle(R.string.no_history_head_label);
         emptyView.setMessage(R.string.no_history_label);
-        swipeActions.detach();
+        swipeActions = new SwipeActions(this, getFragmentTag()).attachTo(recyclerView);
+        swipeActions.setFilter(getFilter());
```

Comment:
I don't think creating a new SwipeActions object is needed here. You can just remove the `detach` line.